oak/server/rust/oak_runtime/src/node/grpc/io.rs

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use log::{debug, error};
+use prost::Message;
+
+use oak_abi::{
+    proto::oak::encap::{GrpcRequest, GrpcResponse},
+    ChannelReadStatus, OakStatus,
+};
+
+use crate::{runtime::RuntimeProxy, Handle};
+
+/// Wrapper for a [`crate::Handle`] that is responsible for sending messages to an Oak channel.
```

Comment:
I don't think these names need to be fully qualified, they are already imported by the `use`s above.